projects/packages/my-jetpack/CHANGELOG.md

```diff
@@ -400,6 +409,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Created package
 
+[0.6.13]: https://github.com/Automattic/jetpack-my-jetpack/compare/0.6.12...0.6.13
```

Comment:
```suggestion
[1.0.0]: https://github.com/Automattic/jetpack-my-jetpack/compare/0.6.12...1.0.0
```

Reply:
Ah we should make the "linked version number" error more descriptive. It yelled at me that the version wasn't linked to anything, but I didn't look further down the doc.